generators/ruby/faraday/templates/README.mustache

```diff
@@ -0,0 +1,104 @@
+
+```ruby
+require {{{gemName}}}
+
+Onfido.configure do |config|
+  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.region = config.region[:EU]
+end
+
+onfido_api = Onfido::DefaultApi.new
+```
+
+All resources share the same interface when making API calls. Use `.create` to create a resource, `.find` to find one, and `.all` to fetch all resources.
+
+### Making a call to the API
+
+```ruby
+applicant = onfido_api.create_applicant(
+  first_name: 'Test',
+  last_name: 'Applicant'
+)
+```
+
+Documentation and code examples can be found at https://documentation.onfido.com
+
+## Error Handling
+
+There are 3 classes of errors raised by the library, all of which subclass `Onfido::OnfidoError`:
+
+- `Onfido::RequestError` is raised whenever Onfido returns a `4xx` response
+- `Onfido::ServerError` is raised whenever Onfido returns a `5xx` response
+- `Onfido::ConnectionError` is raised whenever a network error occurs (e.g., a timeout)
+
+All 3 error classes provide the `response_code`, `response_body`, `json_body`, `type` and `fields` of the error (although for `Onfido::ServerError` and `Onfido::ConnectionError` the last 3 are likely to be `nil`).
+
+```ruby
+def create_applicant
+  {{{gemName}}}.applicant.create(params)
+rescue Onfido::RequestError => e
+  e.type          # => 'validation_error'
+  e.fields        # => { "email": { "messages": ["invalid format"] } }
+  e.response_code # => '422'
+end
+```
+
+### Webhook event verification
+
+Webhook events payload needs to be verified before it can be accessed. Library allows to easily decode the payload and verify its signature before returning it as an object for user convenience:
+
+```ruby
+  require 'onfido/webhook_event_verifier'
+  verifier = Onfido::WebhookEventVerifier.new("_ABC123abc...3ABC123_")
+
+  signature = "a0...760e"
+
+  event = verifier.read_payload('{"payload":{"r...3"}}', signature)
+```
+
+## Contributing
+
+This library is automatically generated using [OpenAPI Generator](https://openapi-generator.tech) - version: {{generatorVersion}};
+therefore all the contributions, except tests files, should target [Onfido OpenAPI specification repository](https://github.com/onfido/onfido-openapi-spec/tree/master) instead of this repository.
+
+For contributions to the tests instead, please follow the steps below:
+
+1. [Fork](<https://github.com/{{ gitUserId }}/{{ gitRepoId }}/fork>) repository
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Make your changes
+4. Commit your changes (`git commit -am 'Add some feature'`)
+5. Push to the branch (`git push origin my-new-feature`)
+6. Create a new Pull Request
+
+## More documentation
+
+More documentation and code examples can be found at <{{ documentationUrl }}>.
+
+## Support
+
+Should you encounter any technical issues during integration, please contact Onfido's Customer Support team
+via the [{{ supportDescription }}]({{ supportUrl }}) which also include support documentation.
```

Comment:
Just to align with changes in [PR#67](https://github.com/onfido/onfido-openapi-spec/pull/67):
 ```suggestion
This library is automatically generated using [OpenAPI Generator](https://openapi-generator.tech) - version: {{generatorVersion}}; therefore all the contributions, except tests files, should target [Onfido OpenAPI specification repository](https://github.com/onfido/onfido-openapi-spec/tree/master) instead of this repository.

For contributions to the tests instead, please follow the steps below:

1. [Fork](<https://github.com/{{ gitUserId }}/{{ gitRepoId }}/fork>) repository
2. Create your feature branch (`git checkout -b my-new-feature`)
3. Make your changes
4. Commit your changes (`git commit -am 'Add some feature'`)
5. Push to the branch (`git push origin my-new-feature`)
6. Create a new Pull Request

## More documentation

More documentation and code examples can be found at <{{ documentationUrl }}>.

## Support

Should you encounter any technical issues during integration, please contact Onfido's Customer Support team
via the [{{ supportDescription }}]({{ supportUrl }}) which also includes support documentation.
```